source/wp-content/themes/wporg-developer/reference/template-source.php

```diff
@@ -15,9 +15,20 @@
 	<section class="source-content">
 		<h2><?php _e( 'Source', 'wporg' ); ?></h2>
 		<p>
-			<?php printf( __( 'File: %s', 'wporg' ),
-				'<a href="' . esc_url( get_source_file_archive_link( $source_file ) ) . '">' . esc_html( $source_file ) . '</a>'
-			); ?>
+			<?php
+			printf(
+				__( 'File: %s.', 'wporg' ),
+				esc_html( $source_file )
```

Comment:
What do you think about wrapping this in `code`?

<img width="527" alt="" src="https://user-images.githubusercontent.com/541093/171662013-891c5f12-e3bb-4380-975c-eb549d4e6303.png">

```suggestion
				'<code>' . esc_html( $source_file ) . '</code>'
```

Reply:
Yep, that looks good! Thanks.